### YamlMime:PythonClass
uid: uamqp.errors.ClientMessageError
name: ClientMessageError
fullName: uamqp.errors.ClientMessageError
module: uamqp.errors
inheritances:
- uamqp.errors.MessageException
constructor:
  syntax: ClientMessageError(exception, info=None)
  parameters:
  - name: exception
    isRequired: true
  - name: info
    defaultValue: None
methods:
- uid: uamqp.errors.ClientMessageError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: uamqp.errors.ClientMessageError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: uamqp.errors.ClientMessageError.args
  name: args
